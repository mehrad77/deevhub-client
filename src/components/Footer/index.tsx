import React from 'react';
import styled from 'styled-components';
import { theme } from 'styled-tools';
import { breakpoints } from '../../utils ';
import { Flex, Box, Text, Contianer } from '..';

const StyledFooter = styled.footer`
  background: ${theme('colors.thisdGrey')};
  height: 100px;
`;

const StyledBox = styled(props => <Box {...props} />)`
  padding: 15px;
  @media ${breakpoints.phoneOnly} {
    padding: 5px;
    &.hideOnMobile {
      display: none;
    }
  }
`;

const Footer = () => {
  return (
    <StyledFooter className="Footer">
      <Contianer>
        <Flex
          flexDirection={['column', 'row']}
          height="100%"
          alignItems="center"
          justifyContent="space-between"
        >
          <StyledBox>
            <Text color="white">Â© DeevHub Copyright 2019</Text>
          </StyledBox>
          <Box className="hideOnMobile" flexGrow={1}></Box>
          <StyledBox>
            <Text color="white">Privacy Policy</Text>
          </StyledBox>
          <StyledBox>
            <Text color="white">Terms & Conditions</Text>
          </StyledBox>
        </Flex>
      </Contianer>
    </StyledFooter>
  );
};

export default Footer;
