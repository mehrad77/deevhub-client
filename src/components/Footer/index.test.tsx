import React from 'react';
import { render } from 'enzyme';
import Footer from '.';

describe('Foote', () => {
  it('Renders Footer', () => {
    const footer = render(<Footer />);
    expect(footer).toMatchSnapshot();
  });
});
