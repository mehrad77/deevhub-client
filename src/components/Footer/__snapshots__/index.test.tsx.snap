// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Foote Renders Footer 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "sc-VigVT kSGxCS Footer",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "sc-gqjmRU hMbKvK",
        },
        "children": Array [
          Object {
            "attribs": Object {
              "class": "sc-ifAKCX sc-EHOje sc-bZQynM rzVkd",
              "height": "100%",
            },
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "sc-bwzfXH SQDzB",
                      "color": "white",
                    },
                    "children": Array [
                      Object {
                        "data": "Â© DeevHub Copyright 2019",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                      "color": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                      "color": undefined,
                    },
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "class": "sc-ifAKCX sc-gzVnrw cmcJb hideOnMobile",
                  },
                  "children": Array [],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "attribs": Object {
                      "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-bwzfXH SQDzB",
                          "color": "white",
                        },
                        "children": Array [
                          Object {
                            "data": "Privacy Policy",
                            "next": null,
                            "parent": [Circular],
                            "prev": null,
                            "type": "text",
                          },
                        ],
                        "name": "span",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                          "color": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                          "color": undefined,
                        },
                      },
                    ],
                    "name": "div",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": Object {
                      "attribs": Object {
                        "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-bwzfXH SQDzB",
                            "color": "white",
                          },
                          "children": Array [
                            Object {
                              "data": "Terms & Conditions",
                              "next": null,
                              "parent": [Circular],
                              "prev": null,
                              "type": "text",
                            },
                          ],
                          "name": "span",
                          "namespace": "http://www.w3.org/1999/xhtml",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                          "x-attribsNamespace": Object {
                            "class": undefined,
                            "color": undefined,
                          },
                          "x-attribsPrefix": Object {
                            "class": undefined,
                            "color": undefined,
                          },
                        },
                      ],
                      "name": "div",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "class": "sc-ifAKCX sc-gzVnrw cmcJb hideOnMobile",
                },
                "children": Array [],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "sc-bwzfXH SQDzB",
                        "color": "white",
                      },
                      "children": Array [
                        Object {
                          "data": "Privacy Policy",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                    },
                  ],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "attribs": Object {
                      "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-bwzfXH SQDzB",
                          "color": "white",
                        },
                        "children": Array [
                          Object {
                            "data": "Terms & Conditions",
                            "next": null,
                            "parent": [Circular],
                            "prev": null,
                            "type": "text",
                          },
                        ],
                        "name": "span",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                          "color": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                          "color": undefined,
                        },
                      },
                    ],
                    "name": "div",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "sc-bwzfXH SQDzB",
                        "color": "white",
                      },
                      "children": Array [
                        Object {
                          "data": "Â© DeevHub Copyright 2019",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                    },
                  ],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "sc-bwzfXH SQDzB",
                      "color": "white",
                    },
                    "children": Array [
                      Object {
                        "data": "Privacy Policy",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                      "color": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                      "color": undefined,
                    },
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "attribs": Object {
                    "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "sc-bwzfXH SQDzB",
                        "color": "white",
                      },
                      "children": Array [
                        Object {
                          "data": "Terms & Conditions",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                    },
                  ],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "sc-ifAKCX sc-gzVnrw cmcJb hideOnMobile",
                  },
                  "children": Array [],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                    },
                    "children": Array [
                      Object {
                        "attribs": Object {
                          "class": "sc-bwzfXH SQDzB",
                          "color": "white",
                        },
                        "children": Array [
                          Object {
                            "data": "Â© DeevHub Copyright 2019",
                            "next": null,
                            "parent": [Circular],
                            "prev": null,
                            "type": "text",
                          },
                        ],
                        "name": "span",
                        "namespace": "http://www.w3.org/1999/xhtml",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "tag",
                        "x-attribsNamespace": Object {
                          "class": undefined,
                          "color": undefined,
                        },
                        "x-attribsPrefix": Object {
                          "class": undefined,
                          "color": undefined,
                        },
                      },
                    ],
                    "name": "div",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              Object {
                "attribs": Object {
                  "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                },
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "sc-bwzfXH SQDzB",
                      "color": "white",
                    },
                    "children": Array [
                      Object {
                        "data": "Terms & Conditions",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "span",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                      "color": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                      "color": undefined,
                    },
                  },
                ],
                "name": "div",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                  },
                  "children": Array [
                    Object {
                      "attribs": Object {
                        "class": "sc-bwzfXH SQDzB",
                        "color": "white",
                      },
                      "children": Array [
                        Object {
                          "data": "Privacy Policy",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "text",
                        },
                      ],
                      "name": "span",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                        "color": undefined,
                      },
                    },
                  ],
                  "name": "div",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "sc-ifAKCX sc-gzVnrw cmcJb hideOnMobile",
                    },
                    "children": Array [],
                    "name": "div",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": Object {
                      "attribs": Object {
                        "class": "sc-ifAKCX sc-gzVnrw kkMFAX sc-jTzLTM kPZgTv",
                      },
                      "children": Array [
                        Object {
                          "attribs": Object {
                            "class": "sc-bwzfXH SQDzB",
                            "color": "white",
                          },
                          "children": Array [
                            Object {
                              "data": "Â© DeevHub Copyright 2019",
                              "next": null,
                              "parent": [Circular],
                              "prev": null,
                              "type": "text",
                            },
                          ],
                          "name": "span",
                          "namespace": "http://www.w3.org/1999/xhtml",
                          "next": null,
                          "parent": [Circular],
                          "prev": null,
                          "type": "tag",
                          "x-attribsNamespace": Object {
                            "class": undefined,
                            "color": undefined,
                          },
                          "x-attribsPrefix": Object {
                            "class": undefined,
                            "color": undefined,
                          },
                        },
                      ],
                      "name": "div",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
            ],
            "name": "div",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {
              "class": undefined,
              "height": undefined,
            },
            "x-attribsPrefix": Object {
              "class": undefined,
              "height": undefined,
            },
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "footer",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
